src/Bigcommerce/Api/Resources/ProductOption.php

```diff
@@ -10,10 +10,13 @@
  */
 class ProductOption extends Resource
 {
+    protected $fieldMap = array(
+        'option' => 'option_id'
+    );
 
 	public function option()
 	{
-		return self::getResource('/options/' . $this->option_id, 'Option');
+		return Client::getResource('/options/' . $this->option_id, 'Option');
```

Comment:
Possibly--I'll double-check in the morning. There was some funkiness with them because some of the resources have actual properties and most have properties on `$this->fields`.